config/default.js

```diff
@@ -126,4 +126,74 @@ module.exports = {
     U: BigInt(5), // non square in Fp
   },
   MAX_QUEUE: 5,
+  ENVIRONMENTS: {
+    mainnet: {
+      name: 'Mainnet',
+      chainId: 1,
+      clientApiUrl: '',
+      optimistApiUrl: '',
+      optimistWsUrl: '',
+      web3WsUrl: '',
+    },
+    ropsten: {
+      name: 'Ropsten',
+      chainId: 3,
+      clientApiUrl: 'https://client1.testnet.nightfall3.com',
+      optimistApiUrl: 'https://optimist1.testnet.nightfall3.com',
+      optimistWsUrl: 'wss://optimist1-ws.testnet.nightfall3.com',
+      web3WsUrl: 'wss://ropsten1-ws.testnet.nightfall3.com',
```

Comment:
Can we remove web3WsUrl from code?

Reply:
It has been removed in last commit `fix: remove testnet Ropsten URL from config`